go/cmd/dolt/commands/diff.go

```diff
@@ -864,6 +864,8 @@ func getTableSchemaAtRef(queryist cli.Queryist, sqlCtx *sql.Context, tableName s
 	return sch, createStmt, nil
 }
 
+// schemaFromCreateTableStmt returns a schema for the CREATE TABLE statement given
+// TODO: this is substantially incorrect, doesn't handle primary key ordering, probably other things too
```

Comment:
This probably bothers me the most in this PR. Is this func silently breaking things?

Reply:
Yes, we are for sure printing schema diffs wrong. Will add bats tests on my next pass